src/consumer/mod.rs

```diff
@@ -1,5 +1,9 @@
 use crate::error::Error;
-use crate::proto::{self, Client, ClientOptions, HeartbeatStatus, Reconnect};
+
+use crate::proto::{
```

Comment:
Heh, yeah, there's a lot of debate around exactly how these should be grouped. My current take is that without perfect automatic grouping separation, the best way forward is to just group everything together and sort alphabetically. After all, the import list isn't something people usually end up looking at manually.

I don't feel too strongly about trying to enforce this in CI since, as you say, the options are currently unstable. But would be okay with adding this as an additional step to the existing `cargo fmt` job if you feel strongly about it :+1:

Reply:
I believe a helper command plus an extra CI check would be nice to have, but definitely not in this PR: I ran `cargo +nightly fmt -- --config group_imports=one` on the sources and some of the modules got formatter - this would be a distracting diff.